Assets/Scripts/CollisionAudio.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+///<summary> 
+/// This class is used to trigger an audio sound whenever the object collides with something.
+///</summary> 
+public class CollisionAudio : MonoBehaviour
+{
+    [Tooltip("Audio OneShot to be played durring a collision event.")] [SerializeField] private AudioClip collisionAudio = null;
+
+
+    private AudioSource audioSource;
+    private Rigidbody rigidBody;
+    private float startTime;
+
+
+    void Start()
+    {
+        try
+        {
+            rigidBody = GetComponent<Rigidbody>();
+            audioSource = GetComponent<AudioSource>();
+        }
+        catch (Exception e)
+        {
+            Debug.LogError("There is not Rigidbody and/or AudioSource component on the game object.\n" + e);
+        }
+        audioSource.mute = true;
+        startTime = Time.time;
+    }
+    void OnCollisionEnter(Collision collision)
+    {
+        float timeSinceStart = Time.time - startTime;
+        if (timeSinceStart > 2) {
+            audioSource.mute = false;
```

Comment:
Why will the audio mute after 2 seconds?

Reply:
The game calculates a ton of collisions when a scene loads. This is stops the player's ears from bleeding because of that